### PR TITLE
updated lib

### PR DESCRIPTION
**Pull request by matthewoates**

### CODE REVIEW

File: lib/cloverCloud.js [hunk 1]
```diff
@@ -35,6 +35,9 @@ function Clover(configuration) {
     this.numberOfDiscoveryMessagesToSend = 10;
 
     this.configuration = configuration;
+    if(!this.configuration) {
```
Comment (samschaack) [inline lib/cloverCloud.js]: TYPO :)

Comment (matthewoates) [inline lib/cloverCloud.js]: This is from remote-pay-cloud

Comment (samschaack) [inline lib/cloverCloud.js]: :cloud: typo

Comment (samschaack) [inline lib/cloverCloud.js]: :smile:

**Pull request merged**